dataStores/chat/ChatController.js

```diff
@@ -0,0 +1,32 @@
+const storage = require('../dataStores/Storage');
+
+class ChatController {
+
+	getMessages(req, res) {
+		return res.status(200).json({messages: storage.messages});
+	}
+
+	saveMessage(req, res) {
+		storage.messages.push(req.body.username);
+		return res.status(200).send();
```

Comment:
Zapis to dałbym kod 201
https://httpstatuses.com/201

Reply:
Ja nie wiem czy warto wchodzić w te kody, które nie są powszechnie znane. Często się spotykałeś z kodem 201?

Reply:
https://gist.github.com/vkostyukov/32c84c0c01789425c29a

Reply:
Jeżeli chcemy się rozwodzić nad tymi kodami to można wstawić te "zaawansowane", jak nie będziemy o tym wspominać to IMO lepiej zostawić te dobrze znane

Reply:
Ja bym został przy podstawowych kodach.